src/modules/bids-asks/actions/translate-binary-orders.js

```diff
@@ -0,0 +1,34 @@
+import { BUY, SELL } from 'modules/transactions/constants/types'
+
+export const translateBinaryOrders = (binaryMarketIds, orders) => {
```

Comment:
I'm a little confused about the purpose of this function.  Is it correct that the UI indexes outcomes starting from 1, but the backend indexes outcomes starting from 0, and therefore this translate function is needed?  Unless it is not feasible for some reason, I would strongly recommend resolving the inconsistency instead (i.e., have the UI use 0-based indexing).

Reply:
this is specifically for open order on outcome 0. Saw this with the canned markets orders. The UI doesn't store outcome 0 on binary markets and won't let users create 'NO' orders only Yes orders. 

This translate method is to visualize/show in UI open orders on outcome 0. That might be created via api.

Reply:
The UI should not display open orders on outcome 0.  It should only display outcome 1.  Users should not be able to trade on outcome 0 at all via the UI.

If canned markets is creating orders on outcome 0, that's something we should fix.